src/plugins/georeferencer/qgsmapcoordsdialog.h

```diff
@@ -13,32 +13,100 @@
 #define MAPCOORDSDIALOG_H
 
 #include <QDialog>
+#include <QMouseEvent>
 
 #include "qgsmaptoolemitpoint.h"
+#include "qgssnappingutils.h"
 #include "qgspoint.h"
+#include "qgsvertexmarker.h"
+#include "qgsmapcanvas.h"
 
 #include <ui_qgsmapcoordsdialogbase.h>
 
 class QPushButton;
 
-class QgsGeorefMapToolEmitPoint : public QgsMapToolEmitPoint
+class QgsGeorefMapToolEmitPoint : public QgsMapTool
```

Comment:
I probably didn't change it because that would make the changeset bigger. The tool still emits a point and behaves like the EmitPoint tool; it just doesn't derive from it anymore. If the group would like me to change the name, I can.